marklogic-data-hub-central/ui/src/pages/TilesView.test.tsx

```diff
@@ -236,7 +236,9 @@ describe("Tiles View component tests for Developer user", () => {
     expect(getByText("Save")).not.toBeDisabled();
     fireEvent.click(getByText("Cancel"));
     // test run
-    fireEvent.click(document.querySelector(".accordion-button"));
+    fireEvent.click(document.querySelector(".accordion-button")!);
+    //    let element = document.querySelector(".accordion-button");
```

Comment:
@MSBrignoni we can probably just remove these redundant lines instead of commenting it

Reply:
Thanks!, fixed